lms/djangoapps/instructor/views/data_access_constants.py

```diff
@@ -16,6 +16,21 @@ def __init__(self, query_type, inclusion, entity_id, filtering, entity_name):
         self.entity_name = entity_name
 
 
+class QueryOrigin:
```

Comment:
You might consider named tuples here [1].

Though it does feel odd to be mixing lightweight classes above with dicts below (and tuples even further down).

Given that we're just mapping strings, it'd be nice to do so consistently.

[1] https://docs.python.org/2/library/collections.html